Description: Start zfs-mount.service after zfs-load-module.service, and if zfs
 module was actually loaded. This should allow installation of probert in
 containers, which pulls in zfs-linux utilities.
Author: Dimitri John Ledkov <xnox@ubuntu.com>


--- zfs-linux-0.8.3.orig/etc/systemd/system/zfs-mount.service.in
+++ zfs-linux-0.8.3/etc/systemd/system/zfs-mount.service.in
@@ -7,6 +7,8 @@ After=zfs-import.target
 After=systemd-remount-fs.service
 Before=local-fs.target
 Before=systemd-random-seed.service
+After=zfs-load-module.service
+ConditionPathExists=/sys/module/zfs
 
 [Service]
 Type=oneshot
